excel_import_export/models/xlsx_import.py

```diff
@@ -50,7 +50,7 @@ def get_external_id(self, record):
             ModelData.create(
                 {
                     "name": "{}_{}".format(record._table, record.id),
-                    "module": "excel_import_export",
+                    "module": "__export__",
```

Comment:
`__excel_import_export__` might be better, borrowing the idea from https://github.com/OCA/server-tools/pull/1948#issuecomment-1376286339?

Reply:
You are right, is better set more specific module name, thank you